lms/djangoapps/courseware/toggles.py

```diff
@@ -86,6 +86,23 @@
     WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones_first_section_celebration', __name__
 )
 
+# .. toggle_name: courseware.mfe_progress_milestones_first_discussion_post
+# .. toggle_implementation: ExperimentWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to display a celebration modal on learner completion of their first
+#   discussion post. Supports staged rollout to students for a new micro-frontend-based implementation
+#   of the courseware page.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2020-11-24
+# .. toggle_target_removal_date: None
+# .. toggle_warnings: Also set settings.LEARNING_MICROFRONTEND_URL and ENABLE_COURSEWARE_MICROFRONTEND and
+#   COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES.
+# .. toggle_tickets: AA-303
+COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_FIRST_DISCUSSION_POST = ExperimentWaffleFlag(
+    WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones_first_discussion_post', __name__,
+    num_buckets=3, use_course_aware_bucketing=False,
```

Comment:
Should these be described in the comments I wonder?

Reply:
Yeah probably